txgithub/token.py

```diff
@@ -9,7 +9,8 @@
 
 def createToken(username, password,
                 note, note_url,
-                scopes, baseURL=None):
+                scopes, baseURL=None,
+                _getPage=client.getPage):
```

Comment:
Same comment here about underscores not making arguments private, and this is public API surface. In this case and below, I'm not sure that the added arguments are sensible as part of the api.